src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetInfoService.cs

```diff
@@ -113,32 +113,16 @@ private void PopulateSnippetCaches()
         {
             Debug.Assert(_expansionManager != null);
 
-            // If the expansion manager is an IExpansionManager, then we can use the asynchronous
-            // population mechanism it provides. Otherwise, getting snippet information from the
-            // IVsExpansionManager must be done synchronously on the UI thread.
-
-            // IExpansionManager was introduced in Visual Studio 2015 Update 1, but will be enabled
-            // by default for the first time in Visual Studio 2015 Update 2. However, the platform
-            // still supports returning the IVsExpansionManager if a major problem in the 
-            // IExpansionManager is discovered, so we must continue supporting the fallback.            
-
             var token = _waiter.BeginAsyncOperation(GetType().Name + ".Start");
 
-            var asyncExpansionManager = _expansionManager as IExpansionManager;
-            if (asyncExpansionManager != null)
-            {
-                // Call the asynchronous IExpansionManager API from a background thread
-                Task.Run(() => PopulateSnippetCacheAsync(asyncExpansionManager))
-                    .CompletesAsyncOperation(token);
-            }
-            else
-            {
-                // Call the synchronous IVsExpansionManager API from the UI thread
-                Task.Factory.StartNew(() => PopulateSnippetCacheOnForeground(_expansionManager),
-                    CancellationToken.None,
-                    TaskCreationOptions.None,
-                    ForegroundTaskScheduler).CompletesAsyncOperation(token);                
-            }
+            // In Dev14 Update2+ the platform always provides an IExpansion Manager
```

Comment:
@rchande Can we refer to it as "Visual Studio 2015 Update 2"?  "Dev14" seems too codewordy.